frontend/packages/ux-editor/src/components/index.ts

```diff
@@ -65,20 +66,21 @@ export const componentIcons: IComponentIcon = {
   NavigationBar: 'fa fa-page-navigation',
   NavigationButtons: 'fa fa-button',
   AttachmentList: 'fa fa-attachment',
+  Panel: 'fa fa-paragraph', // TODO make sure to use the correct icon.
 };
-const Header = {
+const Header: IComponent = {
```

Comment:
Is there a reason why all these components that were implicitly an IComponent now have to be explicit?

Reply:
Because we are using TypeScript and objects should be typed. The second reason is because of the documentation: _"Add a new object describing your new component. The object must have type IComponent, and can contain the following properties..."_ Since the documentation is explicitly telling us that it needs to be the type of IComponent I mean it should be explicitly typed.

Reply:
Are you referring to where it is used? You can also use `... as IComponent` in that param.